mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -592,6 +629,59 @@ object GeneralizedLinearRegression extends DefaultParamsReadable[GeneralizedLine
   }
 
   /**
+    * Tweedie exponential family distribution.
+    * The default link for the Tweedie family is the log link.
+    */
+  private[regression] object Tweedie extends Family("tweedie") {
+
+    val defaultLink: Link = Log
+
+    var variancePower: Double = 1.5
```

Comment:
I think the Tweedie implementation needs to be able to access parameters of the GLM, to read off variancePower.

As it is this is a global variable and two jobs would overwrite each others' values.